hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/source/StreamReadMonitoringFunction.java

```diff
@@ -234,6 +235,9 @@ public void monitorDirAndForwardSplits(SourceContext<MergeOnReadInputSplit> cont
             + "---------- consumed to instant: {}\n"
             + "------------------------------------------------------------",
         conf.getString(FlinkOptions.TABLE_NAME), this.issuedInstant);
+    if (result.isEmpty()) {
+      LOG.warn("In the current round-robin stream read, there are no files to be read in these commits.");
```

Comment:
Maybe just "No new instants to read for current run."

Reply:
And this log is redundant with line 221, we should just keep one of them.

Reply:
> And this log is redundant with line 221, we should just keep one of them.

https://github.com/apache/hudi/pull/12166 It can be seen as a reminder for this PR.  meaning that although there are some commits for current run,  actually no data files available to read.

Reply:
Then reports `No new files to read for current run.`

Reply:
> Then reports `No new files to read for current run.`

Done